imageio/core/functions.py

```diff
@@ -76,12 +76,55 @@
 
 from __future__ import absolute_import, print_function, division
 
+from numbers import Number
+import re
+
 import numpy as np
 
 from . import Request, RETURN_BYTES
 from .. import formats
 
 
+MEMTEST_DEFAULT_MIM = "256MB"
+MEMTEST_DEFAULT_MVOL = "1GB"
+
+
+mem_re = re.compile(r"^([\d.]+)\s*([kKMGTPEZY]*i?[bB])?$")
```

Comment:
replace `([\d.]+)` with `(\d+\.?\d*)` to prevent multiple decimal points

Reply:
Possibly use `?` instead of `*` for first character of unit string (although multiple letters will get stopped by the dict lookup later anyway).